nucleus/common/jul-extensions/src/test/java/fish/payara/acme/handler/SysLogHandlerTest.java

```diff
@@ -0,0 +1,96 @@
+ *  and Distribution License("CDDL") (collectively, the "License").  You
+ *  may not use this file except in compliance with the License.  You can
+ *  obtain a copy of the License at
+ *  https://github.com/payara/Payara/blob/master/LICENSE.txt
+ *  See the License for the specific
+ *  language governing permissions and limitations under the License.
+ *
+ *  When distributing the software, include this License Header Notice in each
+ *  file and include the License file at glassfish/legal/LICENSE.txt.
+ *
+ *  GPL Classpath Exception:
+ *  The Payara Foundation designates this particular file as subject to the "Classpath"
+ *  exception as provided by the Payara Foundation in the GPL Version 2 section of the License
+ *  file that accompanied this code.
+ *
+ *  Modifications:
+ *  If applicable, add the following below the License Header, with the fields
+ *  enclosed by brackets [] replaced by your own identifying information:
+ *  "Portions Copyright [year] [name of copyright owner]"
+ *
+ *  Contributor(s):
+ *  If you wish your version of this file to be governed by only the CDDL or
+ *  only the GPL Version 2, indicate your decision by adding "[Contributor]
+ *  elects to include this software in this distribution under the [CDDL or GPL
+ *  Version 2] license."  If you don't indicate a single choice of license, a
+ *  recipient has the option to distribute your version of this file under
+ *  either the CDDL, the GPL Version 2 or to extend the choice of license to
+ *  its licensees as provided above.  However, if you add GPL Version 2 code
+ *  and therefore, elected the GPL Version 2 license, then the option applies
+ *  only if the new code is made subject to such option by the copyright
+ *  holder.
+ */
+
+package fish.payara.acme.handler;
+
+import fish.payara.logging.jul.PayaraLogManager;
+import fish.payara.logging.jul.cfg.PayaraLogManagerConfiguration;
+import fish.payara.logging.jul.cfg.SortedProperties;
+import fish.payara.logging.jul.handler.SyslogHandler;
+import fish.payara.logging.jul.record.EnhancedLogRecord;
+
+import java.util.logging.Level;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+
+/**
+ * @author David Matejcek
+ */
+public class SysLogHandlerTest {
+
+    private static PayaraLogManagerConfiguration originalConfig;
+    private SyslogHandler handler;
+
+    @BeforeAll
+    public static void backup() {
+        originalConfig = PayaraLogManager.getLogManager().getConfiguration();
+        System.out.println("Original configuration: " + originalConfig);
+    }
+
+
+    @AfterEach
+    public void closeHandler() {
+        if (handler != null) {
+            handler.close();
+        }
+        PayaraLogManager.getLogManager().reconfigure(originalConfig);
+    }
+
+
+    /**
+     * Ths test is dumb, it just uses the handler and expects it will not throw an exception.
+     */
+    @Test
+    void standardUsage() throws Exception {
```

Comment:
throws not needed.

Reply:
I add them usually in all tests, because throws gives no information here. JUnit throws Error instances, so mostly generic Exception is a compromise.
JUnit5 has assertThrows, which is useful for testing malfunctions.